content/en/docs/collector/building/authenticator-extension.md

```diff
@@ -59,7 +59,7 @@ service:
 ```
 
 However, an authenticator will need to be referenced by a consuming component to
-be effective. The following example shows the same extension as above, now being
+be effective. The following example shows the same extension above, now being
 used by a receiver named `otlp/auth`:
```

Comment:
You can delete this paragraph. I added some of the content to the previous paragraph.